### PR TITLE
UI: Fix Dark and System themes select list height

### PR DESCRIPTION
**Pull request by TianQiBuTian**
### Description

Before:
before
After:
after
### Motivation and Context

Limit the influence range of padding in `QListView` to fix the select list height.
### How Has This Been Tested?

Open a select list.
### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by Warchamp7** [comment]
The size of QComboBox items is currently smaller on all themes in master. This is likely due to some other change by us, or in Qt itself.

The proposed changes here don't actually have any effect on the size/padding of QComboBox dropdowns as suggested by the screenshots.

**Comment by TianQiBuTian** [comment]
No, in other themes, the option bar has almost no padding.

Also, the select list on the non-setting page is the correct height.


For reference, this is 27.2.4.

File: UI/data/themes/Dark.qss [hunk 1]
```diff
@@ -770,7 +770,7 @@ OBSBasicSettings {
     qproperty-advancedIcon: url(./Dark/settings/advanced.svg);
 }
 
-OBSBasicSettings QListView::item {
+OBSBasicSettings QListView#listWidget::item {
```
Comment (Warchamp7) [inline UI/data/themes/Dark.qss:773]: This should be

```OBSBasicSettings QListWidget::item {}```

to match how the other themes handle this

File: UI/data/themes/System.qss [hunk 2]
```diff
@@ -200,7 +200,7 @@ OBSBasicSettings {
     qproperty-advancedIcon: url(:settings/images/settings/advanced.svg);
 }
 
-OBSBasicSettings QListView::item {
+OBSBasicSettings QListView#listWidget::item {
```
Comment (Warchamp7) [inline UI/data/themes/System.qss:203]: This should be

```OBSBasicSettings QListWidget::item {}```

to match how the other themes handle this

**Review by Warchamp7 — CHANGES_REQUESTED** [review]
Apologies, I see now what happened. There's a typo in Dark and System for handling this padding.

I've left notes with the correct fix

**Pull request merged**